CHANGELOG.md

```diff
@@ -4,6 +4,30 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v12.1.2](https://github.com/voxpupuli/puppet-r10k/tree/v12.1.2) (2023-10-05)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-r10k/compare/v12.1.1...v12.1.2)
+
+**Implemented enhancements:**
```

Comment:
there are enhancements in this release, so please make it a minor release, not a patch release (12.2.0).